### DISCUSSION TITLE
V5 新版默认启用了 LodController，还不能移除。不希望在画布缩放过程中，够保持文本字号不变，要怎么做到之前版本缩放 Canvas 可以一同缩放文本 呢？

### DISCUSSION THREAD

**Discussion by lvren** [antvis/G6#5181] [General]
V5 新版默认启用了 LodController，还不能移除。不希望在画布缩放过程中，够保持文本字号不变，要怎么做到之前版本缩放 Canvas 可以一同缩放文本 呢？

**Comment by lvren** [comment]
有官方的人在吗？是 v5 之后都必须要求 Label 的大小不进行缩放了？必须要固定？

**Reply by Aarebecca** [reply]
后续会支持这个诉求的

**Reply by lvren** [reply]
这个有 PR 或者 Task 嘛，我关注下

**Reply by lvren** [reply]
@Aarebecca 试用了正式版的 V5，也大概读了下文档，没有找到这个特性呢？要怎么才能控制 Label 不缩放？
